t/plugin/clickhouse-logger.t

```diff
@@ -0,0 +1,219 @@
+            local ok, err = plugin.check_schema({user = "default",
+                                                 password = "a",
+                                                 database = "default",
+                                                 logtable = "t",
+                                                 endpoint_addr = "http://127.0.0.1:10420/clickhouse-logger/test"
+                                                 })
+
+            if not ok then
+                ngx.say(err)
+            else
+                ngx.say("passed")
+            end
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 3: auth configure undefined
+--- config
+    location /t {
+        content_by_lua_block {
+            local plugin = require("apisix.plugins.clickhouse-logger")
+            local ok, err = plugin.check_schema({user = "default",
+                                                 password = "a",
+                                                 database = "default",
+                                                 logtable = "t"
+                                                 })
+
+            if not ok then
+                ngx.say(err)
+            else
+                ngx.say("passed")
+            end
+        }
+    }
+--- response_body
+property "endpoint_addr" is required
+
+
+
+=== TEST 4: add plugin on routes
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "clickhouse-logger": {
+                                "user": "default",
+                                "password": "a",
+                                "database": "default",
+                                "logtable": "t",
+                                "endpoint_addr": "http://127.0.0.1:10420/clickhouse-logger/test"
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1982": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/opentracing"
+                }]],
+                [[{
+                    "action":"set",
+                    "node":{
+                        "value":{
+                            "uri":"/opentracing",
+                            "upstream":{
+                                "scheme":"http",
+                                "nodes":{
+                                    "127.0.0.1:1982":1
+                                }
+                            },
+                            "plugins":{
+                                "clickhouse-logger":{
+                                    "batch_max_size":1000,
```

Comment:
```suggestion
                                    "batch_max_size":1,
```
So the report can be triggered in the CI

Reply:
I thought there was a timer that called this callback function periodically and also if the batch_max_size value was reached. But I added a wait of `--- wait: 5` below, it seem doesn'twork.